src/ChatTag/index.tsx

```diff
@@ -43,9 +44,9 @@ function ChatTag({ author, crosspost, referenceGuild }: TagProps) {
       <Styles.Tag className="verified system">{verified} SYSTEM</Styles.Tag>
     );
 
-  if (crosspost) return <Styles.Tag className="server">SERVER</Styles.Tag>;
+  if (crossPost) return <Styles.Tag className="server">SERVER</Styles.Tag>;
 
-  if (author.flags & (1 << 16))
+  if (author.flags !== undefined && (author.flags & (1 << 16)) !== 0)
```

Comment:
Could you make a little helper function or something (or atleast a comment) for transparancy on what this does, isVerifiedBot or something